protocol/contracts/amo/AuraStaking.sol

```diff
@@ -51,6 +51,11 @@ contract AuraStaking is IAuraStaking, TempleElevatedAccess {
         emit SetRewardsRecipient(_recipient);
     }
 
+    function setRewardTokens(address[] memory _rewardTokens) external override onlyElevatedAccess {
+        rewardTokens = _rewardTokens;
```

Comment:
understood - but checking for dups here would be very gassy since it's just an array, and the values might be unsorted.

Since there's no `mapping` in memory, the only way would be to have another `address[] alreadySeen;` but would then need an O(nLogn) since it has to iterate through both

Something like (untested/unoptimised):
```
address[] alreadySeen = address[](_rewardTokens.length);
uint256 numSeen;
for (uint256 i; i < _rewardTokens.length; ++i) {
    for (uint256 j; j < numSeen; j++) {
        if (_rewardTokens[i] == alreadySeen[j]) revert;
    }
    alreadySeen[numSeen] = _rewardTokens[i];
    ++numSeen;
}
```

I don't think it's worth it given it's protected. The worst that can happen is that there's an extra iteration in getReward